src/Qowaiv.Bson.MongoDB/QowaivBsonSerializationProvider.cs

```diff
@@ -14,28 +14,35 @@ public sealed class QowaivBsonSerializationProvider : IBsonSerializationProvider
     [Pure]
     public IBsonSerializer? GetSerializer(Type? type)
     {
-        IBsonSerializer? serializer = null;
-
-        if (GetCandidateType(type) is not { } tp
-            || Serializers.TryGetValue(tp, out serializer))
+        if (GetCandidateType(type) is { } tp)
```

Comment:
I'd personally change this into:

```cs
if (GetCandidateType(type) is not { } tp)
{
    return null;
}

...
<rest of function>
...
```
Without an `else` branch, to limit the complexity.

Reply:
I like to so see the clear flow in my functions. However, in this case you could argue that an early exist is fine. I changed it.